scripts/utils/PostGitCommitStatus.ps1

```diff
@@ -174,4 +174,41 @@ function Get-TestRun {
     }
     $failedTests = $matchingRun.unanalyzedTests
     return $testUrl,$failedTests
+}
+
+function CheckVstsPersonalAccessToken {
+    param(
+        [Parameter(Mandatory = $True)]
+        [string]$VstsPersonalAccessToken
+    )
+    Write-Output 'starting!'
+    $url = "$env:VSTSTESTRUNSRESTAPI$env:BUILD_BUILDID"
+    Write-Host $url
+    $Token = ":$VstsPersonalAccessToken"
+    $Base64Token = [System.Convert]::ToBase64String([char[]]$Token)
+
+    $Headers = @{
+        Authorization = 'Basic {0}' -f $Base64Token;
+    }
+
+    try {
+        # Basic Parsing prevents the need for Internet Explorer availability.
+        $response = Invoke-WebRequest -Uri $url -Method GET -Headers $Headers -UseBasicParsing
```

Comment:
Can you use HEAD instead of GET?

Reply:
No: The remote server returned an error: (405) Method Not Allowed.